src/components/ae-overscreen/ae-overscreen.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="ae-overscreen" :class="[{ shadow }, align]">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ae-overscreen',
+  props: {
+    /**
+     * Applies a linear shadow gradient to the overscreen
+     */
+    shadow: Boolean,
+
+    /**
+     * Align the elements either left or right, default alignment is center.
+     * Available values are: `left, right`
+     */
+    align: {
+      type: String,
+      validator: value => [
+        'left',
+        'center',
+        'right',
+      ].includes(value),
+      default: 'center',
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../styles/globals';
```

Comment:
When I asked of explicit imports in SCSS, I was not mean to copy/paste `@import '../../styles/globals';`, it is really useless. I mean that we should import particular dependencies of a specific component.
So, please use:
```scss
@import '../../styles/variables/colors';
@import '../../styles/variables/stack';
@import '../../styles/globals/functions';
```
in this case.

Reply:
done.